.docksal/docksal.yml

```diff
@@ -10,63 +10,11 @@ services:
       - MIGRATE_DB_ENV
     volumes:
       - ${PROJECT_ROOT}/.docksal/etc/drush:/etc/drush:ro
-    # Expose port and let Storybook attach to it.
-    expose:
-      - 6006
-    labels:
-      - io.docksal.virtual-host=storybook.${VIRTUAL_HOST},storybook.${VIRTUAL_HOST}.*
-      - io.docksal.virtual-port=6006
 
-  # Fake email
-  mail:
+  solr:
+    image: "devkteam/solr:8"
     extends:
       file: ${HOME}/.docksal/stacks/services.yml
-      service: mail
-      
-  # PHPMyAdmin
-  pma:
-    hostname: pma
-    image: phpmyadmin/phpmyadmin
-    environment:
-      - PMA_HOST=db
-      - PMA_USER=root
-      - PMA_PASSWORD=${MYSQL_ROOT_PASSWORD:-root}
-    labels:
-      - io.docksal.virtual-host=pma.${VIRTUAL_HOST},pma.${VIRTUAL_HOST}.*
-    dns:
-      - ${DOCKSAL_DNS1}
-      - ${DOCKSAL_DNS2}
-
-  # Dashboard displaying results from `fin pa11y`.
-  pa11y:
```

Comment:
You're removing Pa11y?

Reply:
doesn't work on m chips

Reply:
grrr

Reply:
Are we removing this forever?  If so, we have artifacts in the test folder , commands in the primary `README.md`, and in the `/docs` folder that will need to be removed.